src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs

```diff
@@ -791,21 +807,25 @@ public void page_with_false_date_in_title()
         [Fact]
         public void post_default_values()
         {
-            fileSystem.AddFile(@"C:\TestSite\_posts\SomeFile.md", new MockFileData(@"---
+            var filename = @"C:\TestSite\SomeFile.md";
```

Comment:
Because the test failed. The test failed because the Posts get populated from _posts, Pages gets populated from anywhere in the site source. The path in the test vectors was not in the _posts folder (which made me wonder why the tests didn't fail earlier...). Instead of wasting hours, i changed it.

I hope I have not misunderstood something.